library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -65,6 +65,10 @@
    * The number of times to loop through the gif animation.
    */
   private int maxLoopCount = LOOP_FOREVER;
+  /**
+   * The index of the last drawn frame.
+   */
+  private int frameIndex = -1;
```

Comment:
I think this should default to 0. We actually pass in the first frame of the GIF into the Drawable, so it should always be at some index in the range 0..frame count -1.